### ISSUE TITLE
Drop down menu on sign in 

### ISSUE THREAD

**Issue by MaratAkhmatnurov** [ianmartorell/meteor-accounts-ui-bootstrap-3#198]
When I'm clicking to "sign in/Join", nothing's going on. Drop down menu doesn't shown

**Comment by advikguha** [comment]
I'm having the same problem. Did you find a fix?

**Comment by pechitook** [comment]
Same here. I'm using the [bootstrap-sass package](https://github.com/englue/meteor-bootstrap3-sass)

**Comment by pechitook** [comment]
So I found a solution. Turns out that somehow bootstrap is not initializing the dropdown. If I manually run `$('.dropdown-toggle').dropdown()` after the page loads, it works.

The following patch makes it work again, but @ianmartorell you should check this out when you have some time to see if you can replicate the error.

``` js
Template._loginButtonsLoggedOutDropdown.rendered = function() {
    $('.dropdown-toggle').dropdown()
}

Template._loginButtonsLoggedInDropdown.rendered = function() {
    $('.dropdown-toggle').dropdown()
}
```

**Comment by mgebundy** [comment]
Thanks @p4bloch 

Easy fix, but regressions make me a sad panda.

**Comment by pechitook** [comment]
Glad it worked for you too @mgebundy!

**Comment by mherb63** [comment]
I have the same issue.  Is there a way to fix this in css?

@p4bloch where did you put your Template code?  I have my nav template in nav.html and tried your solution in nav.js but it didn't make a difference.

![image](https://cloud.githubusercontent.com/assets/1205814/14194037/27bfa7c4-f778-11e5-9d85-3a2fb5d60b03.png)

**Comment by pechitook** [comment]
@mherb63 I placed it in my JS file as well, not sure why it's not working for you. Are you on latest version? I believe the `.rendered` API had some changes over time. You can try this syntax

```
Template.MyTemplate.onRendered(function () {
 // code
});
```

**Comment by mherb63** [comment]
@p4bloch yes, I used onRendered.  This problem only happens for me when the page size is that of an IPhone 6.  The issue appears when running on an XCode simulator.

**Comment by ShakataGaNai** [comment]
Just chiming in on this to note that I've also run into this error. My project was idle for a while so I just upgraded all the packages and to meteor 1.4.0.1. The package versions I have:

twbs:bootstrap               3.3.6
ian:accounts-ui-bootstrap-3  1.2.89

(Note, I used @p4bloch fix and I'm good for now.)

**Comment by joaoeffting** [comment]
Same problem here :/
